src/main/java/io/debezium/connector/yugabytedb/YugabyteDBChangeRecordEmitter.java

```diff
@@ -227,17 +222,14 @@ private Object[] updatedColumnValues(List<ReplicationMessage.Column> columns, Ta
 
         // initialize to unset
 
-        final Set<String> undeliveredToastableColumns = new HashSet<>(schema
-                .getToastableColumnsForTableId(table.id()));
         for (ReplicationMessage.Column column : columns) {
             // DBZ-298 Quoted column names will be sent like that in messages,
             // but stored unquoted in the column names
             final String columnName = Strings.unquoteIdentifierPart(column.getName());
-            undeliveredToastableColumns.remove(columnName);
 
             int position = getPosition(columnName, table, values);
             if (position != -1) {
-                Object value = column.getValue(() -> (BaseConnection) connection.connection(),
+                Object value = column.getValue(() -> null /*(BaseConnection) connection.connection()*/,
```

Comment:
Instead of passing `null`, can the parameter be removed. At least from my code browsing, looks like we can edit `ReplicationMessage`